### PR TITLE
OSDOCS-5749: Rearchitecture based on PerfScale content audit

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-5749

No new content; just a rearchirecture. No QE required.

Redirects established in https://github.com/openshift/openshift-docs/pull/59933

Link to docs previews:

New Optimizations section:
https://59378--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/optimization/optimizing-storage.html

Recommended perfromance and scalability practices topic is now its own folder, breaking down the topics:
https://59378--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.html

Relocated Cluster Monitoring Operator content:
https://59378--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.html#scaling-cluster-monitoring-operator_recommended-infrastructure-practices

### CODE REVIEW

File: scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+[id="recommended-control-plane-practices"]
+= Recommended control plane practices
+include::_attributes/common-attributes.adoc[]
+:context: recommended-control-plane-practices
+
+toc::[]
+
+This topic provides recommended performance and scalability practices practices for control planes in {product-title}.
```
Comment (GroceryBoyJr) [inline scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc:9]:
```suggestion
This topic provides recommended performance and scalability practices for control planes in {product-title}.
```

File: scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+include::modules/master-node-sizing.adoc[leveloffset=+1]
+
+[id="increasing-aws-flavor-size_{context}"]
+=== Selecting a larger Amazon Web Services instance type for control plane machines
+
+If the control plane machines in an Amazon Web Services (AWS) cluster require more resources, you can select a larger AWS instance type for the control plane machines to use.
+
+[NOTE]
+====
+The procedure for clusters that use a control plane machine set is different than the procedure for clusters that do not use a control plane machine set.
```
Comment (GroceryBoyJr) [inline scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc:22]:
```suggestion
The procedure for clusters that use a control plane machine set is different from the procedure for clusters that do not use a control plane machine set.
```

Comment (GroceryBoyJr) [inline scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc:22]:
```suggestion
The procedure for clusters that use a control plane machine set is different from the procedure for clusters that do not use a control plane machine set.
```

File: scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="recommended-etcd-practices"]
+= Recommended etcd practices
+include::_attributes/common-attributes.adoc[]
+:context: recommended-etcd-practices
+
+toc::[]
+
+This topic provides recommended performance and scalability practices practices for etcd in {product-title}.
```
Comment (GroceryBoyJr) [inline scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc:9]:
```suggestion
This topic provides recommended performance and scalability practices for etcd in {product-title}.
```

File: scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="recommended-infrastructure-practices"]
+= Recommended infrastructure practices
+include::_attributes/common-attributes.adoc[]
+:context: recommended-infrastructure-practices
+
+toc::[]
+
+This topic provides recommended performance and scalability practices practices for infrastructure in {product-title}.
```
Comment (GroceryBoyJr) [inline scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc:9]:
```suggestion
This topic provides recommended performance and scalability practices for infrastructure in {product-title}.
```

**Review by GroceryBoyJr — COMMENTED** [review]
@ahardin-rh only a few small nits. Great work!

**Comment by ahardin-rh** [comment]
@GroceryBoyJr Good eye. Fixed! Thanks!

**Comment by ahardin-rh** [comment]
Redirect PR is in place and approved. Merging! :rocket:

**Pull request merged**